src/assets/i18n/en.json5

```diff
@@ -4327,6 +4335,10 @@
 
   "relationships.OrgUnit.isPublicationOfAuthor.Publication": "Authored Publications",
 
+  "relationships.OrgUnit.isPublicationOfContributor.Publication": "Publications",
+
+  "relationships.OrgUnit.isProjectOfFundingAgency.Project": "Research Projects",
```

Comment:
This also needs differentiation from `isProjectOfOrgUnit`.  Maybe it should be something like "Researcher Projects (funder of)"?